lib/js/components/Drawer/DrawerTile/DrawerTile.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-	<ds-tile v-bind="$attrs" class="drawerTile" />
+	<div class="drawerTile">
```

Comment:
now `Tile` props are injected as HTML attributes on this div :(

Reply:
It's not the first place where we do this. I'm not sure if we should care. It's caused by https://vuejs.org/guide/components/attrs.html#attribute-inheritance

Reply:
there are more problems with not defined props on this component... 
Because of `v-bind="$attrs"` when you use `<drawer-tile class="test">` you get this class name applied on both DrawerTile and Tile containers...

Reply:
https://github.com/bethinkpl/design-system/pull/310/commits/7eaf5c38e3d5427e5dcb8807d5a4d714a60ee18e